.gitignore

```diff
@@ -27,3 +27,4 @@ obj/
 [Rr]elease*/
 _ReSharper*/
 [Tt]est[Rr]esult*
+QuickLayout/packages
```

Comment:
I think the whole gitignore could be replaced with a better one. Which ignores all packages.